index.html

```diff
@@ -132,7 +132,9 @@ <h3>Here's a timeline of Dr. Favaloro's life:</h3>
       mentioned, however, that despite the immense costs of bypass surgery, Dr Favaloro operated daily on
       indigent
       patients, something that he felt was both a necessity and his obligation.</li>
-    <li class="h3-fix">Recognitions & distinctions</li>
```

Comment:
Please split  ul  and change li class into a h3 headline title

Reply:
- Please when giving a review, try to explain why the change requested is important. example
`Please remove all unused commented code, so that your code will be cleaner and easy to debug`